packages/launchpad/src/checklist-item/index.tsx

```diff
@@ -2,13 +2,21 @@ import { Badge, Gridicon } from '@automattic/components';
 import { Button } from '@wordpress/components';
 import classnames from 'classnames';
 import { translate, useRtl } from 'i18n-calypso';
+import { FC } from 'react';
```

Comment:
Nit: this should be a type import.
```suggestion
import { type FC } from 'react';
```

Reply:
Fixed here https://github.com/Automattic/wp-calypso/pull/87141/commits/322631506ef06de292a7c037760c8c647804fc8b